### PR TITLE
Use luma.gl in arcgis module

### PR DESCRIPTION
**Pull request by tsherif**
Update arcgis module to use luma.gl instead of raw gl calls as much as possible.

### CODE REVIEW

File: modules/arcgis/src/arcgis-deck-external-renderer.js [hunk 1]
```diff
@@ -38,52 +40,51 @@ export default function loadArcGISDeckExternalRenderer(externalRenderers, Collec
...
     this.deckgl.setProps({
-      layers: this.deckLayers.items.slice(),
       viewState: {
         latitude: this.view.center.latitude,
         longitude: this.view.center.longitude,
-        zoom: this.view.zoom,
-        bearing: this.view.camera.heading,
-        pitch: this.view.camera.tilt
+        zoom: this.view.featuresTilingScheme.scaleToLevel(state.scale),
+        bearing: -state.rotation,
```
Comment (Pessimistress) [inline modules/arcgis/src/arcgis-deck-external-renderer.js]: I don’t think these should be changed...

File: modules/arcgis/src/arcgis-deck-layer-view-2d.js [hunk 2]
```diff
@@ -72,22 +74,18 @@ export default function loadArcGISDeckLayerView2D(BaseLayerViewGL2D) {
...
-        this.program = null;
+      if (this.buffer) {
+        this.buffer.delete();
       }
 
-      if (this.vertexBuffer) {
-        gl.deleteBuffer(this.vertexBuffer);
-        this.vertexBuffer = null;
+      if (this.deckFbo) {
+        this.destroyFramebuffer(this.deckFbo);
```
Comment (Pessimistress) [inline modules/arcgis/src/arcgis-deck-layer-view-2d.js]: Just call `this.deckFbo.delete()`

File: modules/arcgis/src/arcgis-deck-external-renderer.js (hunk 1 above)
Comment (tsherif) [reply modules/arcgis/src/arcgis-deck-external-renderer.js]: Weird, I don't remember changing this. Maybe copy pasta.

**Comment by tsherif** [comment]
@Pessimistress Fixed.

**Pull request merged**